src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/ConsoleParameters.cs

```diff
@@ -34,7 +34,15 @@ public string LogFilePath
                     throw new ArgumentException("LogFilePath must point to a valid directory for logging!");
                 }
 
-                this.logFilePath = value;
+                // Ensure path is double quoted. if path has white space then it can create problem.
```

Comment:
Yes we have take care of it in case of TestHost and Datacollector.
Yes we can do code refactoring to have repeated code in one location.